packages/edit-navigation/src/store/utils.js

```diff
@@ -154,6 +154,10 @@ export function computeCustomizedAttribute(
 				title: block.attributes?.label,
 				original_title: '',
 				url: block.attributes.url,
+				description: block.attributes.description,
+				xfn: block.attributes.rel.split( ' ' ),
+				classes: block.attributes.className.split( ' ' ),
```

Comment:
This throws an error when `rel` or `className` is `undefined`. 

1. Add a new link to the Navigation screen.
2. Click _Save_.
3. Note the _There was an error._ message.

Reply:
Whoops, I'll fix it!

Reply:
Here: #25229